kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -92,7 +93,7 @@ def _call_rpc(packed_args: Tuple[str, KCFG, int]) -> bool:
                     implication_every_block=implication_every_block,
                 )
             except Exception as e:
-                _LOGGER.error(f'Proof crashed: {_cfgid}\n{e}')
+                _LOGGER.error(f'Proof crashed: {_cfgid}\n{e}\n{traceback.format_exc()}')
```

Comment:
Adding this might not be necessary if you specify `exc_info=True` ([docs](https://docs.python.org/3/library/logging.html#logging.debug)). I haven't tried this though, so don't take my word for it.

Reply:
Done.